pkg/test/framework/runtime/components/galley/native.go

```diff
@@ -125,6 +126,35 @@ func (c *nativeComponent) ApplyConfig(yamlText string) (err error) {
 	return
 }
 
+// ApplyConfigDir implements Galley.ApplyConfigDir.
+func (c *nativeComponent) ApplyConfigDir(sourceDir string) (err error) {
+	return filepath.Walk(sourceDir, func(path string, info os.FileInfo, err error) error {
+		targetPath := c.configDir + string(os.PathSeparator) + path[len(sourceDir):]
+		if info.IsDir() {
+			scopes.Framework.Debugf("Making dir: %v", targetPath)
+			os.MkdirAll(targetPath, os.ModePerm)
+		} else {
+			scopes.Framework.Debugf("Copying file to: %v", targetPath)
+			contents, readerr := ioutil.ReadFile(path);
+		  if (readerr != nil) {
+		  	return readerr
+			}
+		  ioutil.WriteFile(targetPath, contents, os.ModePerm)
+		}
+		return nil
+	})
+}
+
+// GetGalleyAddress returns the galley mcp server address
+func (c *nativeComponent) GetGalleyAddress() string {
```

Comment:
Is this needed? I don't see where it's used.

Reply:
Oops, botched merge. Fixed